.github/workflows/make.yml

```diff
@@ -0,0 +1,34 @@
+name: make
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
Nit: why the `'`s for 3.10?

Reply:
if you don't do the `'` it parses it as a float and tries to install python `3.1`

Reply:
[301bdfe](https://github.com/datavaluepeople/phoenix-scrape/pull/2/commits/301bdfea6991c6f14907586b7d7202aae945d65b)